code/_onclick/ai.dm

```diff
@@ -36,20 +35,15 @@
 		return
 
 	var/turf/pixel_turf = get_turf_pixel(A)
-	var/turf_visible
-	if(pixel_turf)
-		turf_visible = GLOB.cameranet.checkTurfVis(pixel_turf)
-		if(!turf_visible)
-			if(istype(loc, /obj/item/device/aicard) && (pixel_turf in view(client.view, loc)))
-				turf_visible = TRUE
-			else
-				if (pixel_turf.obscured)
-					log_admin("[key_name_admin(src)] might be running a modified client! (failed checkTurfVis on AI click of [A]([COORD(pixel_turf)])")
-					message_admins("[key_name_admin(src)] might be running a modified client! (failed checkTurfVis on AI click of [A]([ADMIN_COORDJMP(pixel_turf)]))")
-					if(REALTIMEOFDAY >= chnotify + 9000)
-						chnotify = REALTIMEOFDAY
-						send2irc_adminless_only("NOCHEAT", "[key_name(src)] might be running a modified client! (failed checkTurfVis on AI click of [A]([COORD(pixel_turf)]))")
-				return
+	if(isnull(pixel_turf))
+		return
+	if(!can_see(A))
+		log_admin("[key_name_admin(src)] might be running a modified client! (failed can_see on AI click of [A]([COORD(pixel_turf)]))")
```

Comment:
change `key_name_admin` to `key_name` here, also make it a var so `log_admin` and `send2irc_adminless_only` uses the same thing